test/test/helpers/use-fake-dom/index.js

```diff
@@ -1,32 +1,18 @@
-import debug from 'debug';
+/**
+ * External dependencies
+ */
+import { flow } from 'lodash';
 
-import reactTestEnvSetup from 'lib/react-test-env-setup';
+/**
+ * Internal dependencies
+ */
+import reactTestEnvSetup from 'react-test-env';
 
-const log = debug( 'calypso:test:helpers:setup-dom-env' );
+const useFakeDom = flow( [ reactTestEnvSetup.useFakeDom, () => {
+	after( () => delete require.cache[ require.resolve( 'page' ) ] );
```

Comment:
I'd recommend preserving the comment that described this behavior in the original file.